app/src/main/java/net/gsantner/opoc/util/FileUtils.java

```diff
@@ -541,4 +541,18 @@ public static boolean fileExists(final File checkFile) {
         }
         return false;
     }
+
+    // Get the title of the file
+    public static String extractTitle(final File file) {
+        final String name = file.getName();
+        final int doti = name.lastIndexOf(".");
+        return (doti < 0) ? name : name.substring(0, doti);
+    }
+
+    /// Get the file extension of the file
```

Comment:
getFilenameExtension

Reply:
I feel so too :joy:

This code isn't new though. Just moved here from Document.